client/src/components/shared/Info/Contracts.js

```diff
@@ -1,56 +1,44 @@
 import React from 'react'
 import ExternalLink from '../ExternalLink'
-import {showNumberCurrency} from '../../Notices/utilities'
+import {ShowNumberCurrency} from '../../Notices/utilities'
 import {compose, withState, withHandlers} from 'recompose'
 import ChevronUp from 'react-icons/lib/fa/chevron-up'
 import ChevronDown from 'react-icons/lib/fa/chevron-down'
 import './InfoList.css'
 
-const _Contracts = (({data, toggledOn, toggle}) => {
+const _Contracts = ({data, toggledOn, toggle}) => {
   return [
     <tr key="header" className="clickable" onClick={toggle}>
       <td>
-        <button className="showHideBtn btn btn-link">
-          Zmluvy
-        </button>
+        <button className="showHideBtn btn btn-link">Zmluvy</button>
       </td>
       <td className="sizeCell">
         <strong>{data.length}</strong>
         {toggledOn ? (
-          <ChevronUp
-            className="sizeCellArrow"
-            onClick={toggle}
-            aria-hidden="true"
-          />
+          <ChevronUp className="sizeCellArrow" onClick={toggle} aria-hidden="true" />
         ) : (
-          <ChevronDown
-            className="sizeCellArrow"
-            onClick={toggle}
-            aria-hidden="true"
-          />
+          <ChevronDown className="sizeCellArrow" onClick={toggle} aria-hidden="true" />
         )}
       </td>
     </tr>,
-    toggledOn
-      ? (
-        <tr className="noBorder">
-          <td colSpan="2">
-            <ul className="contractList list-unstyled">
-              {data.map((contract) => (
-                <li key={contract.source}>
-                  <ExternalLink
-                    url={contract.source}
-                    text={[contract.customer, ', ', showNumberCurrency(contract.total)]}
-                  />
-                </li>
-              ))}
-            </ul>
-          </td>
-        </tr>
-      )
-      : null,
+    toggledOn && (
```

Comment:
are you sure this is the same as `toggledOn ? stuff : null` ? wouldn't 'false' string be rendered ? (or react complalining that a node should be jsx or null ?)

Reply:
I though, but I double checked: https://codesandbox.io/s/ynyp67yqx. It kind of makes sense as you can use <React.Fragment> which acts like a normal react component and when you use ```variable && <Component />``` nothing is rendered if variable is false